server/lib/util.js

```diff
@@ -95,3 +96,23 @@ function resolveObject(object) {
       return settled;
     });
 }
+
+/**
+ * @method dateFormatter
+ *
+ * @description
+ * Accepts an object of key/value pairs. Returns the same object with all values
+ * that are dates converted to a standard format.
+ */
+function dateFormatter(Rows, dateFormat) {
+  const DATE_FORMAT = dateFormat || 'YYYY-MM-DD HH:mm:ss';
+  return Rows.map(function (element) {
+    Object.keys(element).forEach(function (currentKey) {
```

Comment:
A cleaner way to do this would be with [lodash](https://lodash.com/docs/4.17.4#forEach).  It allows you to use functional programming on objects.  For example:
```js
function dateFormatter(rows, dateFormat) {
  return rows.map((element) => {
    return _.forEach(element, (value, key) => {
      if (_.isDate(key)) {
         element[key] = moment(value).format(DATE_FORMAT);
      }
    });
  });
}
```